### ISSUE TITLE
eeprom msft violation

### ISSUE THREAD

**Issue by JazzMaster** [xqemu/xqemu#228]
read the file in one byte(ff) at a time from disk- and demand a file before starting the emulator.
here is your patch.

Thanks for the 1.0 eeprom.

xbox.c:

// XBOX_TODO: Should be passed in through configuration -and is illegal to distribute
const uint8_t default_eeprom[] = {};

**Comment by dracc** [comment]
Read for a minute longer and you'll see it's the bunnie eeprom which is provided under a FOSS license.

**Comment by mborgerson** [comment]
Duplicate of #28
